examples/nlp/language_modeling/conf/megatron_bert_config.yaml

```diff
@@ -128,7 +132,7 @@ model:
     #   - /raid/data/pile/my-gpt3_00_text_document
     #   - .5
     #   - /raid/data/pile/my-gpt3_01_text_document
-    data_prefix: ???
```

Comment:
Here as well ?

Reply:
similar to above reason, try to make conversion work easier, since it's directly load this yaml. If put ??? it will raise an error while loading I think